cmd/explaintest/r/tpch.result

```diff
@@ -184,36 +184,36 @@ limit 100;
-    ├─HashLeftJoin_47	155496.00	root	inner join, inner:TableReader_70, equal:[eq(tpch.partsupp.ps_partkey, tpch.part.p_partkey)]
-    │ ├─HashRightJoin_50	8155010.44	root	inner join, inner:HashRightJoin_52, equal:[eq(tpch.supplier.s_suppkey, tpch.partsupp.ps_suppkey)]
-    │ │ ├─HashRightJoin_52	100000.00	root	inner join, inner:HashRightJoin_58, equal:[eq(tpch.nation.n_nationkey, tpch.supplier.s_nationkey)]
-    │ │ │ ├─HashRightJoin_58	5.00	root	inner join, inner:TableReader_63, equal:[eq(tpch.region.r_regionkey, tpch.nation.n_regionkey)]
-    │ │ │ │ ├─TableReader_63	1.00	root	data:Selection_62
-    │ │ │ │ │ └─Selection_62	1.00	cop	eq(tpch.region.r_name, "ASIA")
-    │ │ │ │ │   └─TableScan_61	5.00	cop	table:region, range:[-inf,+inf], keep order:false
-    │ │ │ │ └─TableReader_60	25.00	root	data:TableScan_59
-    │ │ │ │   └─TableScan_59	25.00	cop	table:nation, range:[-inf,+inf], keep order:false
-    │ │ │ └─TableReader_65	500000.00	root	data:TableScan_64
-    │ │ │   └─TableScan_64	500000.00	cop	table:supplier, range:[-inf,+inf], keep order:false
-    │ │ └─TableReader_67	40000000.00	root	data:TableScan_66
-    │ │   └─TableScan_66	40000000.00	cop	table:partsupp, range:[-inf,+inf], keep order:false
-    │ └─TableReader_70	155496.00	root	data:Selection_69
-    │   └─Selection_69	155496.00	cop	eq(tpch.part.p_size, 30), like(tpch.part.p_type, "%STEEL", 92)
-    │     └─TableScan_68	10000000.00	cop	table:part, range:[-inf,+inf], keep order:false
-    └─HashAgg_73	8155010.44	root	group by:tpch.partsupp.ps_partkey, funcs:min(tpch.partsupp.ps_supplycost), firstrow(tpch.partsupp.ps_partkey)
-      └─HashRightJoin_77	8155010.44	root	inner join, inner:HashRightJoin_79, equal:[eq(tpch.supplier.s_suppkey, tpch.partsupp.ps_suppkey)]
-        ├─HashRightJoin_79	100000.00	root	inner join, inner:HashRightJoin_85, equal:[eq(tpch.nation.n_nationkey, tpch.supplier.s_nationkey)]
-        │ ├─HashRightJoin_85	5.00	root	inner join, inner:TableReader_90, equal:[eq(tpch.region.r_regionkey, tpch.nation.n_regionkey)]
-        │ │ ├─TableReader_90	1.00	root	data:Selection_89
-        │ │ │ └─Selection_89	1.00	cop	eq(tpch.region.r_name, "ASIA")
-        │ │ │   └─TableScan_88	5.00	cop	table:region, range:[-inf,+inf], keep order:false
-        │ │ └─TableReader_87	25.00	root	data:TableScan_86
-        │ │   └─TableScan_86	25.00	cop	table:nation, range:[-inf,+inf], keep order:false
-        │ └─TableReader_92	500000.00	root	data:TableScan_91
-        │   └─TableScan_91	500000.00	cop	table:supplier, range:[-inf,+inf], keep order:false
-        └─TableReader_94	40000000.00	root	data:TableScan_93
-          └─TableScan_93	40000000.00	cop	table:partsupp, range:[-inf,+inf], keep order:false
+  └─HashRightJoin_44	155496.00	root	inner join, inner:HashLeftJoin_50, equal:[eq(tpch.part.p_partkey, tpch.partsupp.ps_partkey) eq(tpch.partsupp.ps_supplycost, min(ps_supplycost))]
```

Comment:
Why plan id changed?

Reply:
Because now we can push down property for `order by a asc, b desc`, which will enlarge the search space.